packages/account/src/Components/api-token/api-token.scss

```diff
@@ -50,6 +50,7 @@
         grid-template-columns: 1fr 1fr;
         grid-template-rows: 1fr 1fr;
         grid-gap: 1.6rem;
+        padding-bottom: 11px;
```

Comment:
We try not use `px` but `rem` in Deriv app.
why is it `11px`? design is displaying `16px`?

Reply:
I will change it to rem.
Regarding the 11px, according to the design the text element that is under the Step 1 (the title of Step 2) has a height of 20px and the padding above it is 16px. However, in production the text element has a height of 25px (including the height of h2 (21px) and the margin of the container (4px) above it) that's why it seemed reasonable to add the padding of 11px instead of 16px. Should I change it to 16px (1.6rem)?

Reply:
Change according to the design so if it is 11px then 1.1rem it is

Reply:
Got it. Thank you :)